dbt/models/iasworld/schema/iasworld.owndat.yml

```diff
@@ -286,27 +264,3 @@ sources:
               tags:
                 # Currently too many failures for this signal to be useful
                 - test_qc_exclude_from_workbook
-          - expression_is_true:
-              name: iasworld_owndat_addr1_address_is_well_formed
-              expression: |
-                CASE
-                  WHEN LENGTH(REGEXP_EXTRACT(addr1, '^[N|S|E|W]{0,2}\s{0,1}([0-9]{1,5})',1)) <= 5 THEN TRUE
-                  WHEN REGEXP_LIKE(UPPER(addr1), '^[PO BOX|P\.O\. BOX|P O BOX|BOX|P O BX|PO BX|PO BX|P.O.BOX]') THEN TRUE
-                  WHEN REGEXP_LIKE(UPPER(addr1), '^[C\/O|ONE|TWO|THREE|FOUR|FIVE|SIX|SEVEN|EIGHT|NINE]') THEN TRUE
-                  ELSE FALSE
-                END
```

Comment:
I couldn't really figure out how to translate this to a `legdat` test, given that there's no equivalent of `owndat.addr1` on `legdat` and I can't quite tell what these tests are trying to glean. There's already a version of the first case statement defined on `legdat` (`iasworld_legdat_adrno_length_lte_5`), and the other two statements just seem to be covering alternative formats that can be true if `addr1` doesn't start with the address number, which doesn't really make much sense under a more normalized schema. I'm open to suggestions if you have ideas for how to adapt this, though!

Reply:
Makes sense, thanks for the extra context!